include/UnifiedCvo/utils/CvoPointCloud.hpp

```diff
@@ -129,17 +131,26 @@ namespace cvo {
     void erase(size_t index);
 
     // getters
+    std::vector<cvo::CvoPoint> get_points() const {return points_;}
+    cvo::CvoPoint point_at(unsigned int index) const {return points_[index];}
     int num_points() const {return num_points_;}
     int size() const {return num_points_;}
     int num_classes() const {return num_classes_;}
     int num_features() const {return feature_dimensions_;}
     int feature_dimensions() const {return feature_dimensions_;}
     int num_geometric_types() const {return num_geometric_types_; }
-    const std::vector<Eigen::Vector3f> & positions() const {return positions_;}
+    const std::vector<Eigen::Vector3f> positions() const {
```

Comment:
is this positions() still used?

Reply:
Yes, still in this version. Though it's better to replace this getter in all implementations I think. I will try on spare time.